final_prep/README.md

```diff
@@ -23,52 +23,52 @@ Some things to consider as you refactor include...
+- [X] I have refactored my `section1` exercises to the best of my ability
+- [X] I have refactored my `section2` exercises to the best of my ability
+- [X] I have refactored my `section3` exercises to the best of my ability
 
 ## Gearing Up for Gear Ups
 
-Gear Up is a part of our curriculum where we talk about topics such as privilege and power, and ethical issues in the tech industry. 
+Gear Up is a part of our curriculum where we talk about topics such as privilege and power, and ethical issues in the tech industry.
 
-- [ ] [Complete this assignment](https://github.com/turingschool/gear-up/blob/main/m0/Intro.To.GearUp.md)
-  - [ ] Add a link for your gist here: 
+- [X] [Complete this assignment](https://github.com/turingschool/gear-up/blob/main/m0/Intro.To.GearUp.md)
+- [X] [link to gist](https://gist.github.com/lcole37/3e5b3cb083fe97ff76879d5d64de3f10)
 
 ## Time Management Prep
 
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
 
-- [ ] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Back End curriculum site](https://backend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
-- [ ] Study/Project work time (usually about 25 hours/week outside of class time)
-- [ ] Personal, Health + Wellness
+- [X] Standard Mod 1 class schedule (find the Mod 1 calendar [on the Back End curriculum site](https://backend.turing.edu/)). The detailed classes held on your actual start date may not yet be posted; you can look back at the previous inning to get a feel for what Weeks 1-3 will be like, but know it's not guaranteed to be exactly the same. Also, as you already know, class is held from 9-12 and 1-4MT, Monday-Friday.
+- [X] Study/Project work time (usually about 25 hours/week outside of class time)
+- [X] Personal, Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+-  <img width="1333" alt="Screen Shot 2022-02-13 at 12 45 56 AM" src="https://user-images.githubusercontent.com/98673086/153744214-923961b7-8cac-4206-a3e5-0b1d76d96d0d.png">
+- <img width="1333" alt="Screen Shot 2022-02-13 at 12 46 08 AM" src="https://user-images.githubusercontent.com/98673086/153744218-ebf322a0-a074-468b-aa58-1f359a4c26e8.png">
+- <img width="1333" alt="Screen Shot 2022-02-13 at 12 46 22 AM" src="https://user-images.githubusercontent.com/98673086/153744227-267473c6-5122-48ec-98e8-8f88bd9f0f3d.png">
```

Comment:
Nice start with your calendars. I understand the Mod 1 calendar wasn't updated when you worked on this, but when it is be sure to revisit your Mod 1 calendar at that time. I'd recommend making some of your study/work times more specific: what lesson are you reviewing/previewing during that time? What do you hope to accomplish?